packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.tsx

```diff
@@ -36,7 +36,7 @@ export const DetailsRowCheck = (props: IDetailsRowCheckProps) => {
   const isPressed = props.isSelected || props.selected;
 
   return (
-    <div
+    <button
```

Comment:
The problem with it being a `<button>` is that it becomes focusable, when designers have specifically asked that it not be. The toggle-selection element for keyboarding is the row itself. The checkbox is a visual aid for mouse-targeting only.

Reply:
Any recommendations on how to fix the attached Bug w/o using `<button>`?

Reply:
Also, as far as I know, any interactive element *should* be focus-able and Design should know this.